### PR TITLE
Implement edit admin

### LINKED ISSUE #301: add Edit for Admins

### PR DESCRIPTION
**Pull request by EllisFinnOsen**
Add's the edit admin feature. also only displays accessLevel Super and Admin on the "admins" page.
closes #301

### CODE REVIEW

File: client/src/components/pages/admins/adminFields.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+export const Categories = {
+    admin: { id: 'admin', title: 'Admin Info' }
+};
+
+const memberFields = [
```
Comment (karlmolina) [inline client/src/components/pages/admins/adminFields.js]: should be adminFields

File: client/src/components/pages/admins/ViewAllAdmins.js [hunk 2]
```diff
@@ -15,7 +15,8 @@ export class ViewAllAdmins extends Component {
 
     componentDidMount() {
         // call redux action to retrieve all admins from api
-        this.props.adminActions.filter();
+        this.props.adminActions.filter({ accessLevel: { $ne: 'Volunteer' } });
```
Comment (karlmolina) [inline client/src/components/pages/admins/ViewAllAdmins.js]: remove these changes I already fixed this, I changed it in the action definition instead

File: client/src/components/pages/members/AddMember.js [hunk 3]
```diff
@@ -0,0 +1,371 @@
+import React, { Component } from 'react';
```
Comment (karlmolina) [inline client/src/components/pages/members/AddMember.js]: why must you add this file

**Review by karlmolina — CHANGES_REQUESTED** [review]

File: client/src/components/pages/members/AddMember.js (hunk 3 above)
Comment (EllisFinnOsen) [reply client/src/components/pages/members/AddMember.js]: your guess is as good as mine

File: client/src/assets/stylesheets/pages/addMember.scss [hunk 4]
```diff
@@ -0,0 +1,13 @@
+@import '_presets.scss';
```
Comment (karlmolina) [inline client/src/assets/stylesheets/pages/addMember.scss]: remove this file too

**Review by karlmolina — CHANGES_REQUESTED** [review]

File: client/src/components/pages/admins/ViewAllAdmins.js [hunk 5]
```diff
@@ -16,6 +16,7 @@ export class ViewAllAdmins extends Component {
     componentDidMount() {
         // call redux action to retrieve all admins from api
         this.props.adminActions.filter();
+        //this.props.adminActions.filter({ accessLevel: 'Super' });
```
Comment (karlmolina) [inline client/src/components/pages/admins/ViewAllAdmins.js]: remove this comment

**Review by karlmolina — CHANGES_REQUESTED** [review]

**Review by karlmolina — APPROVED** [review]

**Pull request merged**